docs/development/upgrading-node.md

```diff
@@ -38,7 +38,8 @@ So in short, the primary steps are:
 
 ## Updating Electron's Node [fork](https://github.com/electron/node)
 
-1. Create a branch in https://github.com/electron/node: `electron-node-vX.X.X`
+1. Ensure that `master` on `electron/node` has updated release tags from `nodejs/node`
+1. Create a branch in https://github.com/electron/node: `electron-node-vX.X.X` where the base that you're branching from is the tag for the desired update
```

Comment:
I guess markdown renumbers this anyway, but FWIW there are two number 1s here :)